l3.c

```diff
@@ -1,3 +1,14 @@
+/**
+ * *****************************************************************************
+ * \file l3.c
+ * \author Greg Law
+ * \brief L3: Lightweight Logging Library Implementation
+ * \version 0.1
+ * \date 2023-12-24
+ *
+ * \copyright Copyright (c) 2023
```

Comment:
Should the Copyright messages in diff files change to: 2023-2024? 

That kind of annual update is a nightmare ... How _does_ one manage that in other oss repos?